draft-irtf-cfrg-frost.md

```diff
@@ -737,6 +700,14 @@ such as messages or the set of individual commitments. Note that the Coordinator
 is *not* trusted with any private information and communication at the time of signing
 can be performed over a public but reliable channel.
 
+The protocol as specified in this document does not target the following goals.
+
+* Post quantum security. FROST requires the hardness of the Discrete Logarithm Problem.
```

Comment:
```suggestion
* Post quantum security. FROST, like generic Schnorr signatures, requires the hardness of the Discrete Logarithm Problem.
```

Reply:
I only make this comment because the Shamir secret sharing is in fact quantum resilient, but the group actions like the additive group stuff to generate the group Schnorr signature, is not